test/testcafe/framework/shared/index.js

```diff
@@ -1,4 +1,9 @@
-import { t } from 'testcafe';
+import { t, ClientFunction } from 'testcafe';
+
+export const renderWidget = ClientFunction((settings) => {
+  // function `renderPlaygroundWidget` is defined in playground/main.js
+  window.renderPlaygroundWidget(settings);
```

Comment:
Probably not necessary, but is good practice to ensure this function is present on the `window` object before running.